celestia/chain.json

```diff
@@ -34,28 +34,25 @@
   },
   "codebase": {
     "git_repo": "https://github.com/celestiaorg/celestia-app",
-    "recommended_version": "v1.13celestiaorg/cosmos-sdk v1.24.1-sdk-v0.46.16.0",
+    "recommended_version": "v2.1.2",
```

Comment:
This fixes a regression introduced in https://github.com/cosmos/chain-registry/pull/5297 because there is no version of celestia-app named "v1.13celestiaorg/cosmos-sdk v1.24.1-sdk-v0.46.16.0"